@xen-orchestra/xapi/sr.mjs

```diff
@@ -30,7 +30,7 @@ class Sr {
       host,
       device_config,
       physical_size,
-      name_label,
+      name_label: name_label.trim(),
```

Comment:
Just a reminder, the card is about names & descriptions.

Another approach may have been to make `Editable/Text` trim value by default.

Reply:
I am not really a fan of changing all the editable fields behaviour. 
I updated the PR with description, and fixing a typo